internal/bminventory/inventory.go

```diff
@@ -1510,6 +1511,69 @@ func (b *bareMetalInventory) GetClusterDefaultConfig(_ context.Context, _ instal
 	return installer.NewGetClusterDefaultConfigOK().WithPayload(&body)
 }
 
+func getPlatformFromVendor(vendor models.SystemVendor) models.PlatformType {
+	var supportedManufacturers = map[string]models.PlatformType{
```

Comment:
should probably be a const?

Reply:
You mean global? I think map cannot be const

Reply:
it's use only in this function so it's not a must (it can be a var) but this is fine as well.

Reply:
> Why SNO is not supported?

This is how the task was defined